src/app/src/components/ResetPasswordForm.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react';
+import { arrayOf, bool, string } from 'prop-types';
```

Comment:
I hit a compile error. Fixed it by changing the prop-types import to
```
import { arrayOf, bool, func, string } from 'prop-types';
```

Reply:
Ah, this was my mistake. Thanks for catching. I fixed this in 
baaa7b9